kube/services/k8resources/gro/gro-rc.yaml

```diff
@@ -42,14 +42,15 @@ spec:
               add_header X-Frame-Options "SAMEORIGIN" always;
               add_header X-Content-Type-Options "nosniff" always;
               add_header X-XSS-Protection "1; mode=block" always;
+              proxy_cache on;
+              proxy_cache_revalidate on;
+              proxy_cache_lock on;
```

Comment:
This makes sure we only send one request back to the app at a time and any subsequent requests should wait on the first and then serve from cache.